packages/component-library/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@fogo/component-library",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "engines": {
+    "node": ">=22.14.0"
+  },
+  "exports": {
+    "./Link": {
+      "default": "./dist/Link/index.mjs"
+    },
+    "./package.json": "./package.json",
+    "./theme": "./dist/theme.scss"
+  },
+  "scripts": {
+    "build": "ts-duality --noCjs",
+    "build:scss": "copyfiles -u 1 \"src/**/*.scss\" ./dist",
+    "build:storybook": "storybook build",
+    "build:svg": "copyfiles -u 1 \"src/**/*.svg\" ./dist",
```

Comment:
I think you're safe to omit this, we don't have svgs (there's so few custom svgs here that for the cases we do have -- basically just the logo -- I just made them react components instead of using svgr)